src/js/tracker.js

```diff
@@ -1954,21 +1946,15 @@
 		/**
 		 * @param number seed The seed used for MurmurHash3
 		 */
-		apiMethods.setUserFingerprintSeed = function(seed) {
-			helpers.warn('setUserFingerprintSeed is deprecated. Instead add a "userFingerprintSeed" field to the argmap argument of newTracker.');
-			configUserFingerprintHashSeed = seed;
-			userFingerprint = detectors.detectSignature(configUserFingerprintHashSeed);
+		apiMethods.setUserFingerprintSeed = function() {
+			helpers.warn('User Fingerprinting is no longer supported. This function will be removed in a future release.');
 		};
 
 		/**
 		 * Enable/disable user fingerprinting. User fingerprinting is enabled by default.
-		 * @param bool enable If false, turn off user fingerprinting
 		 */
-		apiMethods.enableUserFingerprint = function(enable) {
-			helpers.warn('enableUserFingerprintSeed is deprecated. Instead add a "userFingerprint" field to the argmap argument of newTracker.');
-			if (!enable) {
-				userFingerprint = '';
-			}
+		apiMethods.enableUserFingerprint = function() {
+			helpers.warn('User Fingerprinting is no longer supported. This function will be removed in a future release.');
```

Comment:
This string 'User Fingerprinting....' appears three times in the minified bundle.  You could minify further if the string was a constant, which you reused on lines 1834, 1950, 1957.

Reply:
Opened #791 to sort this out for all repeated strings